pom.xml

```diff
@@ -549,24 +549,6 @@
                 </plugins>
             </build>
         </profile>
-        <profile>
-            <id>jdk11</id>
-            <properties>
-                <scala.plugin.version>4.6.1</scala.plugin.version>
-                <maven.compiler.release>11</maven.compiler.release>
-                <maven.compiler.source>11</maven.compiler.source>
-                <maven.compiler.target>11</maven.compiler.target>
-            </properties>
-        </profile>
-        <profile>
-            <id>jdk17</id>
-            <properties>
-                <scala.plugin.version>4.6.1</scala.plugin.version>
-                <maven.compiler.release>17</maven.compiler.release>
-                <maven.compiler.source>17</maven.compiler.source>
-                <maven.compiler.target>17</maven.compiler.target>
-            </properties>
-        </profile>
```

Comment:
I have a reviewed change for the CI repo removing the explicit profile activation from CI jobs. 

However, you can convince yourself that this PR is not breaking the CI as is by executing Maven with one of these profiles. The build will succeed with the warning  

> The requested profile "jdkWhatever" could not be activated because it does not exist.

We don't  yet enforce profiles in the enforcer plugin.